examples/playground/src/deck-with-gmaps.js

```diff
@@ -0,0 +1,109 @@
+/* global console, document, window */
+import React, {Component} from 'react';
+import {GoogleMapsOverlay} from '@deck.gl/google-maps';
+
+const HOST = 'https://maps.googleapis.com/maps/api/js';
+const LOADING_GIF = 'https://upload.wikimedia.org/wikipedia/commons/d/de/Ajax-loader.gif';
+
+const style = {
+  height: 1000,
+  width: 1000
+};
+
+function injectScript(src) {
+  return new Promise((resolve, reject) => {
+    const script = document.createElement('script');
+    script.src = src;
+    script.addEventListener('load', resolve);
+    script.addEventListener('error', e => reject(e.error));
+    document.head.appendChild(script);
+  });
+}
+
+function deepEqual(x, y) {
+  const ok = Object.keys;
+  const tx = typeof x;
+  const ty = typeof y;
+  return x && y && tx === 'object' && tx === ty
+    ? ok(x).length === ok(y).length && ok(x).every(key => deepEqual(x[key], y[key]))
+    : x === y;
+}
+
+function loadGoogleMapApi(apiKey, onComplete) {
+  const url = `${HOST}?key=${apiKey}&libraries=places`;
+  injectScript(url)
+    .then(() => onComplete())
+    // eslint-disable-next-line no-console
+    .catch(e => console.error(e));
+}
+
+export default class DeckWithGoogleMaps extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      gmapsLoaded: window.google !== undefined
```

Comment:
There are a lot of google libraries that define window.google. But I assume it is not an issue in this app.

Type out googleMaps

Reply:
Thus you might want to check for
```js
window.google && window.google.maps
```
?